packages/account/src/Sections/Profile/LanguageSettings/__tests__/language-settings.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import LanguageSettings from '../language-settings';
+
+jest.mock('react-i18next', () => ({
+    ...jest.requireActual('react-i18next'),
+    useTranslation: jest.fn(() => ({ i18n: { changeLanguage: jest.fn() } })),
+}));
+
+jest.mock('@deriv/translations', () => {
+    const original_module = jest.requireActual('@deriv/translations');
+    return {
+        ...original_module,
+        getAllowedLanguages: jest.fn(() => ({ lang_1: 'Test Lang 1', lang_2: 'Test Lang 2' })),
+    };
+});
+
+describe('LanguageSettings', () => {
+    const mockRootStore = mockStore({
+        common: {
+            changeSelectedLanguage: jest.fn(),
```

Comment:
```suggestion
```
It's the default value in the `mockStore` 🤔 I think we can omit it from here 🤔

Reply:
@farzin-deriv , yes. sure will delete it here